MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/IEnggDiffractionView.h

```diff
@@ -343,7 +351,23 @@ class IEnggDiffractionView {
  *
  * @return bool
  */
-  virtual bool saveOutputFiles() const = 0;
+  virtual bool saveFocusedOutputFiles() const = 0;
+
+  /**
+  * Produces vanadium curves graph with three spectrum for calib
+  * output. Runsm plotSpectrum function via python.
```

Comment:
typo: 'Runsm'

Maybe the detail of how it is implemented should not be given in the documentation of the interface. Strictly speaking that's specific to the Qt view.